RecoTracker/LSTCore/interface/alpaka/LST.h

```diff
@@ -0,0 +1,102 @@
+#ifndef RecoTracker_LSTCore_interface_alpaka_LST_h
+#define RecoTracker_LSTCore_interface_alpaka_LST_h
+
+#include "RecoTracker/LSTCore/interface/alpaka/Common.h"
+#include "RecoTracker/LSTCore/interface/LSTESData.h"
+
+#include <cstdlib>
+#include <numeric>
+#include <alpaka/alpaka.hpp>
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
+  class LSTEvent;
+
+  class LST {
+  public:
+    LST() = default;
+
+    void run(Queue& queue,
+             bool verbose,
+             LSTESData<Device> const* deviceESData,
+             std::vector<float> const& see_px,
+             std::vector<float> const& see_py,
+             std::vector<float> const& see_pz,
+             std::vector<float> const& see_dxy,
+             std::vector<float> const& see_dz,
+             std::vector<float> const& see_ptErr,
+             std::vector<float> const& see_etaErr,
```

Comment:
I'm not sure I understand the comment: all variables passed as arguments are used in the code.
 Is the concern that fewer variables can be passed since they can be recomputed on the fly? `Err`s are not represented by anything else, they can not be derived from something else.

Reply:
No, what I meant is that errors do not follow the reference system of the rest of variables,  pt and eta can be computed from px, py, pz sure, but I assumed one reference system was enough